src/main/java/org/openstreetmap/atlas/utilities/command/subcommands/ShardToBoundsCommand.java

```diff
@@ -0,0 +1,101 @@
+package org.openstreetmap.atlas.utilities.command.subcommands;
+
+import java.util.List;
+
+import org.openstreetmap.atlas.geography.sharding.SlippyTile;
+import org.openstreetmap.atlas.utilities.command.abstractcommand.AbstractAtlasShellToolsCommand;
+import org.openstreetmap.atlas.utilities.command.abstractcommand.CommandOutputDelegate;
+import org.openstreetmap.atlas.utilities.command.abstractcommand.OptionAndArgumentDelegate;
+import org.openstreetmap.atlas.utilities.command.parsing.ArgumentArity;
+import org.openstreetmap.atlas.utilities.command.parsing.ArgumentOptionality;
+import org.openstreetmap.atlas.utilities.command.terminal.TTYAttribute;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author lcram
+ */
+public class ShardToBoundsCommand extends AbstractAtlasShellToolsCommand
+{
+    private static final Logger logger = LoggerFactory.getLogger(ShardToBoundsCommand.class);
+
+    private static final String INPUT_SHARDS = "shards";
+
+    private final OptionAndArgumentDelegate optargDelegate;
```

Comment:
optargDelegate?

Reply:
`optionAndArgumentDelegate`. I know we have a no abbreviation rule but I have been abbreviating it since the name is so darn long.

Reply:
nit: optargDelegate -> optionAndArgumentDelegate